parse/Tokens.h

```diff
@@ -310,7 +310,9 @@
     (Population)                                \
     (PopulationCenter)                          \
     (Position)                                  \
+    (Precision)                                 \
     (PreferredFocus)                            \
+    (PreferredPrey)                             \
```

Comment:
Rename to `PriorityTargets` or similar. "Prey" is not really applicable.

Reply:
ok, that was from a prototype before. i was looking for nomenclature from hunting to differentiate between things you would like to shoot, a thing which is already in your scope depending if you want to shoot it or not. So i tried to say the thing you want to shoot is prey. And things in your scope are targets. PriorityTargets works for me.

Reply:
(in the prototype you could query the targets PreyType and compare with the PreferredPreyCondition, but since i switched to the flexible FOCS conditions it was not necessary anymore to assign a PreyType)

Reply:
There is a small part in the code where the prey type stuff is applicable - the part where the planet defenses construct the condition to prefer shooting at ships. I'll change prey to priorityTargets everywhere else.